src/startStopScene.ts

```diff
@@ -69,6 +69,27 @@ export async function startScene(lineStart?: number) {
         lineNumber--;
     }
 
+    // If current line contains: "def construct(" without "#" before it,
+    // then we insert a special comment line, and move the cursor to it
+    // (because `manimgl -se <lineNumber>` doesn't work on the "def construct(" line)
+    let idx = lines[lineNumber].indexOf("def construct(");
+    if (idx !== -1) {
+        if (lines[lineNumber].slice(0, idx).includes("#")) {
+            idx = -1;
+        }
+    }
+    if (idx !== -1) {
+        lineNumber++;
+        const HELPER_LINE = '        #';
```

Comment:
These are some strong assumptions you make here:
- that the editor is using 4 spaces per indentation level (it could use 2 spaces instead, or 42, or tabs)
- that the `construct()` method is defined inside a class that is defined at the root level (Python classes can be defined inside classes)

Another strategy could be to put this line at the same indentation level and with the same spacing used as the subsequent line inside the `construct()` method (if there is any).

---

Also I'd prefer that the extension is not making any edits in my file, if there's a solution that works without this comment line. E.g. what just came to my mind is copying to the clipboard buffer with this line added, but not actually making this change in the editor. It just feels a bit invasive to me. Ah no, copying won't work since we're just passing the filename to Manim... Ok, then maybe create a temporary file? No, that'd probably be overkill.

---

Furthermore, use camelCase for local variables. Or alternatively define this with SNAKE_CASE at the top of this file with a docstring and e.g. a more expressive name like `COMMENT_INSIDE_CONSTRUCT_METHOD`.

Reply:
because of these concerns
(not wanting to edit the user's file)
I'll be closing this PR for now